tests/yam/validate/validate_dud.pm

```diff
@@ -25,6 +25,13 @@ sub run {
     } else {
         die "Error, JSON profile in DUD file not loaded";
     }
+
+    my $kernel_dud_output = script_output("journalctl | grep dud");
```

Comment:
should be just `$journalcl_output`, but how do we know is this dud and not the other dud which produce this output?